tests/integration/dynamic_ip_test.py

```diff
@@ -589,6 +527,52 @@ def test_dhcp_on_bridge0(dhcpcli_up_with_delay_status):
     assert origin_ipv6_state == new_ipv6_state
 
 
+def test_slave_ipaddr_learned_via_dhcp_added_as_static_to_linux_bridge(
+    dhcpcli_up
+):
+    dhcpcli_up[Interface.KEY][0][Interface.IPV4] = create_ipv4_state(
+        enabled=True, dhcp=True
+    )
+
+    libnmstate.apply(dhcpcli_up)
+
+    slave_ifname = dhcpcli_up[Interface.KEY][0][Interface.NAME]
+    current_state = statelib.show_only((slave_ifname,))
+    client_current_state = current_state[INTERFACES][0]
+    dhcpcli_ip = client_current_state[Interface.IPV4][InterfaceIPv4.ADDRESS]
+
+    bridge_state = create_bridge_subtree_state()
+    bridge_state = add_port_to_bridge(bridge_state, slave_ifname)
+
+    ipv4_state = create_ipv4_state(enabled=True, dhcp=False)
+    ipv4_state[InterfaceIPv4.ADDRESS] = dhcpcli_ip
+    bridge_iface_state = {Interface.IPV4: ipv4_state}
+    with linux_bridge(
+        'brtest0', bridge_state, bridge_iface_state, create=False
```

Comment:
wouldn't it be 'better' to also indicate the 'extra_iface_state' keyword argument there ? why only for the 'create' parameter?